### PR TITLE
Fixed broken links in README

### LINKED ISSUE #2934: Broken hyperlink: Read this and decide which version you wish to install
#### Preliminaries

- [ ] I have followed the latest version of the
      [installation instructions](https://docs.manim.community/en/stable/installation.html).
- [ ] I have checked the [troubleshooting page](https://docs.manim.community/en/stable/installation/troubleshooting.html) and my problem is either not mentioned there,
      or the solution given there does not help.

## Description of error

following text has [link] in the readme.md file that is not working:
Read [this](https://docs.manim.community/en/stable/installation/versions.html) and decide which version you wish to install

## Installation logs

<details><summary>Terminal output</summary>

```
PASTE HERE OR PROVIDE LINK TO https://pastebin.com/ OR SIMILAR
```

</details>

## System specifications

<details><summary>System Details</summary>

- OS (with version, e.g., Windows 10 v2004 or macOS 10.15 (Catalina)):
- RAM:
- Python version (`python/py/python3 --version`):
- Installed modules (provide output from `pip list`):
```
PASTE HERE
```
</details>

<details><summary>LaTeX details</summary>

+ LaTeX distribution (e.g. TeX Live 2020):
+ Installed LaTeX packages:

</details>

<details><summary>FFMPEG</summary>

Output of `ffmpeg -version`:

```
PASTE HERE
```
</details>

### PR DESCRIPTION
**Pull request by geophree**
## Overview: What does this pull request change?
This PR corrects two link in the main repo's `README.md`.
- Fix the link to the description of the differences between 3b1b and community versions of manim. 
- Fix the link to the list of command-line arguments.

Fixes #2934

## Reviewer Checklist
- [ ] The PR title is descriptive enough for the changelog, and the PR is labeled correctly
- [ ] If applicable: newly added non-private functions and classes have a docstring including a short summary and a PARAMETERS section
- [ ] If applicable: newly added functions and classes are tested

### CODE REVIEW

**Comment by geophree** [comment]
I'm 99% sure I found the right pages to link to, but someone more familiar with the project should double-check.

versions: https://docs.manim.community/en/stable/faq/installation.html
CLI args: https://docs.manim.community/en/stable/guides/configuration.html

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Manim is an animation engine for explanatory math videos. It's used to create pr
 
 ## Installation
 
-> **WARNING:** These instructions are for the community version _only_. Trying to use these instructions to install [3b1b/manim](https://github.com/3b1b/manim) or instructions there to install this version will cause problems. Read [this](https://docs.manim.community/en/stable/installation/versions.html) and decide which version you wish to install, then only follow the instructions for your desired version.
+> **WARNING:** These instructions are for the community version _only_. Trying to use these instructions to install [3b1b/manim](https://github.com/3b1b/manim) or instructions there to install this version will cause problems. Read [this](https://docs.manim.community/en/stable/faq/installation.html) and decide which version you wish to install, then only follow the instructions for your desired version.
```
Comment (behackl) [inline README.md:38]: Good point! Perhaps it makes sense to link directly to the correct section.

```suggestion
> **WARNING:** These instructions are for the community version _only_. Trying to use these instructions to install [3b1b/manim](https://github.com/3b1b/manim) or instructions there to install this version will cause problems. Read [this](https://docs.manim.community/en/stable/faq/installation.html#why-are-there-different-versions-of-manim) and decide which version you wish to install, then only follow the instructions for your desired version.
```

**Review by behackl — APPROVED** [review]
Left a minor suggestion, LGTM otherwise, please have a look. Thanks for your contribution!

**Comment by geophree** [comment]
Committed your suggestion.

**Pull request merged**